src/platform-includes/session-replay/install/javascript.capacitor.mdx

```diff
@@ -1,7 +1,17 @@
-```bash {tabTitle: Yarn}
-yarn add @sentry/capacitor @sentry/replay
+Install the Sentry Capacitor SDK alongside the corresponding Sentry SDK for the framework you're using, such as Angular in this example:
+
+```bash {tabTitle:Angular}
+# npm
+npm install --save @sentry/capacitor @sentry/angular-ivy @sentry/replay
+
+# yarn
+yarn add @sentry/capacitor @sentry/angular-ivy @sentry/replay
 ```
```

Comment:
does the user need to install `@sentry/replay` if they're already doing `@sentry/angular-ivy` or `@sentry/vue` or whatever the framework is?

Reply:
cc @lucas-zimerman ^